README.md

```diff
@@ -30,281 +30,19 @@
     <a href="https://www.youtube.com/channel/UCjf2teVEuYVvvVC-gFZNq6w"><img src="https://img.shields.io/badge/youtube-subscribe-fc1c1c.svg?style=flat-square"></a>
 </p>
 
-# surrealdb.go
-
-The official SurrealDB SDK for Go.
-
-## Documentation
-
-View the SDK documentation on [pkg.go.dev](https://pkg.go.dev/github.com/surrealdb/surrealdb.go) or [here](https://surrealdb.com/docs/integration/libraries/golang).
-
 ## How to install
 
 ```sh
 go get github.com/surrealdb/surrealdb.go
 ```
 
-## Getting started
-
-To connect to SurrealDB and perform data operations, please refer to the complete example in [./example/main.go](./example/main.go).
-
-> This example requires SurrealDB to be [installed](https://surrealdb.com/install) and running on port 8000.
-
-### Running the Example
-
-The easiest way to get started is to run the example directly:
-
-```sh
-# Clone the repository
-git clone https://github.com/surrealdb/surrealdb.go.git
-cd surrealdb.go
-
-# Run the example
-go run ./example
-```
-
-The `./example` directory contains a complete working example demonstrating basic CRUD operations.
-
-### Testable Examples
-
-Testable example files (`example*_test.go`) demonstrate various SDK features including:
-- Query operations and transactions
-- Relations and graph traversal
-- Bulk operations (insert, upsert)
-- Authentication methods
-- Custom CBOR configuration
-- And many more use cases
-
-You can run any of the example tests with:
-```sh
-go test -v ./ -run ExampleName
-```
-
-If you're viewing this documentation at `pkg.go.dev`, you can view the examples alongside corresponding SDK functions.
-
-## Executing SurrealQL
-
-### Using Query
```

Comment:
Moved to the `surrealdb` package doc and to the `surrealdb.Query` doc.

surrealdb package: https://pkg.go.dev/github.com/surrealdb/surrealdb.go#section-documentation
Query: https://pkg.go.dev/github.com/surrealdb/surrealdb.go#Query